xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPIGL.cpp

```diff
@@ -75,9 +83,20 @@ bool CRendererVAAPI::Configure(const VideoPicture &picture, float fps, unsigned
   interop.glEGLImageTargetTexture2DOES = (PFNGLEGLIMAGETARGETTEXTURE2DOESPROC)eglGetProcAddress("glEGLImageTargetTexture2DOES");
   interop.eglDisplay = CRendererVAAPI::m_pWinSystem->GetEGLDisplay();
 
+  bool useVaapi2{}, hevc{};
+  VAAPI::CVaapi2Texture::TestInterop(CRendererVAAPI::m_pWinSystem->GetVADisplay(), CRendererVAAPI::m_pWinSystem->GetEGLDisplay(), useVaapi2, hevc);
```

Comment:
VADisplay in carried in CVaapiRenderPicture

Reply:
fixed